Update Missing Models (#147)
ie3-institute/simona: docs/readthedocs/about.md

(about)=
# About SIMONA

## Overview

SIMONA is an agent-based discrete-event power system simulation model developed at the [Institute of Energy Systems, Energy Efficiency and Energy Economics](https://ie3.etit.tu-dortmund.de) at [TU Dortmund University](https://www.tu-dortmund.de), Germany.

## Further information and Contact Details

> Please visit SIMONA Website under [https://simona.ie3.e-technik.tu-dortmund.de](https://simona.ie3.e-technik.tu-dortmund.de) for further Information. There you also find actual contact details. 

Reviewer: ```suggestion
Please visit SIMONA Website at [https://simona.ie3.e-technik.tu-dortmund.de](https://simona.ie3.e-technik.tu-dortmund.de) for further Information. There you also find contact details of current developers. 
```
